recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -25,6 +25,11 @@ if "%CI%" == "" (
     echo - %CI% >> .ci_support\%CONFIG%.yaml
 )
 
+:: Remove chocolatey gfortran.
+if "%CI%" == "azure" (
+  rm "C:\ProgramData\Chocolatey\bin\gfortran.exe"
```

Comment:
Can we remove `C:\ProgramData\Chocolatey\bin\` from path?

Reply:
Probably. I'm not sure how to manipulate paths on Windows though.